src/enums.rs

```diff
@@ -2280,4 +2282,19 @@ pub mod key {
     pub const Prev_VMode: Key = gdk_sys::GDK_KEY_Prev_VMode as u32;
     pub const LogWindowTree: Key = gdk_sys::GDK_KEY_LogWindowTree as u32;
     pub const LogGrabInfo: Key = gdk_sys::GDK_KEY_LogGrabInfo as u32;
+
+    pub trait Keys {
+        fn to_unicode(self) -> Option<char>;
+        fn name(self) -> Option<GString>;
+    }
+
+    impl Keys for Key {
```

Comment:
Why not `impl Key` (and `pub fn`s) here?

I think `Key` is also just a type alias: as part of this it should become a newtype wrapper (`pub struct Key(u32)`, deriving clone, copy, partialeq, eq, partialord, ord, hash at least plus impl `FromGlib` / `ToGlib`).

Reply:
I created the trait because the rust compiler told me to do that when I tried to `impl Key`, but as you said I tried with the type alias, if we convert to a newtype that should work.

Reply:
I've added a new commit trying to implement the functionality this way, but I'm not sure if it's correct what I'm doing, and with this new commit the API changes so any other crate that uses `gdk::enum::keys` will be broken.